node/sifrr-server/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:12.7.0
+
+RUN npm -g install pm2
+
+WORKDIR /usr/src/app
+
+COPY app.js package.json ./
+
+RUN npm install
+
+ENV NODE_CLUSTER_SCHED_POLICY=rr
+ENV NODE_ENV production
+
+CMD pm2-runtime start app.js -i $(nproc)
```

Comment:
`uWebSockets.js` is not compatible with `node` **12**

Reply:
however @jkyberneees run if on `node` **12** (with a wrapper)  in https://github.com/the-benchmarker/web-frameworks/pull/1570

Reply:
Hi @waghanza, I think I have to verify that, here I run it with node v10 https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/JavaScript/0http/0http.dockerfile, have you run a successful iteration with node 12?

Reply:
Well, I can confirm uWebSockets.js v15.11 works on my Mac using Node v12.7!

Reply:
strange, the binary seems to exists for both `darwin` and `linux` => https://github.com/uNetworking/uWebSockets.js/tree/v15.11.0
but made on **travis** for `osx` and **appveyor** for `linux`

Reply:
sifrr with updated uwebsockets is not yet released, I will release and try to update here as well.

Reply:
I have know 
```javascript
Error: Cannot find module 'graphql'
```

Reply:
when trying to launch on container

Reply:
getting connection timeout now

Reply:
works on travis though